packages/site/pages/components/appframe.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable react/display-name */
+import React from 'react'
+
+import {
+  Chrome,
+  Code,
+  Content,
+  Intro,
+  P,
+  PageHeading,
+  PropTypes,
+  SectionHeading
+} from '../../src/ui/index.js'
+
+export default _ => (
+  <Chrome>
+    <Content title="AppFrame">
+      <PageHeading packageName="appframe">App Frame</PageHeading>
+      <Intro>
+        The App Frame component provides page structure and integrates with
+        other navigation components.
+      </Intro>
+
+      <br />
+
+      <P>Install the component dependency:</P>
+      <Code language="bash">
+        npm install @pluralsight/ps-design-system-appframe
+      </Code>
+
+      <P>Include a React component in your project:</P>
+      <Code language="javascript">
+        import AppFrame from '@pluralsight/ps-design-system-appframe'
+      </Code>
+
+      <PropTypes
+        props={[
+          PropTypes.row(['children', <code>node</code>, false, null]),
+          PropTypes.row([
+            'sidenav',
+            <>
+              <code>node</code>
+              <span> | </span>
+              <code>{'({ visible  }) => node'}</code>
+            </>,
+            false,
+            null,
+            'Area to display secondary navigation'
+          ]),
+          PropTypes.row([
+            'sidenavOpen',
+            <code>boolean</code>,
+            false,
+            null,
+            'Control if the sidenav container is opened'
+          ]),
+          PropTypes.row([
+            'onRequestSideNavClose',
+            'Event => ()',
+            null,
+            null,
+            'Callback triggered when the sidenav should be closed. Use if controlling the open state'
+          ]),
+          PropTypes.row([
+            'onRequestSideNavOpen',
+            'Event => ()',
+            null,
+            null,
+            'Callback triggered when the sidenav should be opened. Use if controlling the open state'
+          ]),
+          PropTypes.row([
+            'topnav',
+            <>
+              <code>node</code>
+              <span> | </span>
+              <code>
+                {'({ closeSidenav, openSidenav, sidenavOpen }) => node'}
+              </code>
+            </>,
+            false,
+            null,
+            'Area to display primary navigation - such as a NavBar'
+          ])
+        ]}
+      />
+
+      <SectionHeading>Usage</SectionHeading>
+      <br />
+
+      <Code language="html">
```

Comment:
A live preview would be nice. Even a diagram of where the thing lives on the page, even something like the one here https://github.com/ps-dev/prism

I could see that picture answer a lot of the questions of what it looks like, where it appears, why you'd want it on your page.

Reply:
yeah i like the picture idea. i'll add one real quick. also i think i'll add a codesandbox once the package is published - it's not a great candidate for an inline preview because it really needs a whole viewport to show off